### ISSUE TITLE
[doc] typos, updating

### ISSUE THREAD

**Issue by muzimuzhi** [T-F-S/tcolorbox#108]
Just a place for recording typos found in doc from time to time. I will keep the ordering `1., 2., 3., ...` so maintainer will know which one is already fixed in source, which one is newly reported. (Should I open a new issue every time?) If acceptable, I can provide pull request directly as well.

1. `skins` library
`the \meta{options} are is appended.` -> `the \meta{options} are appended.`
https://github.com/T-F-S/tcolorbox/blob/55d9deed444eef58c65a88b893500255f44ac7ba/doc/latex/tcolorbox/tcolorbox.doc.skins.tex#L705-L706

1. (multiple places) `skins` library, left single quotation marks in subsection titles
![image](https://user-images.githubusercontent.com/6376638/89717469-82020800-d9e9-11ea-965e-d3bd480d89ad.png)

`\subsection{Skin Family 'standard'}` -> ``\subsection{Skin Family `standard'}``
https://github.com/T-F-S/tcolorbox/blob/55d9deed444eef58c65a88b893500255f44ac7ba/doc/latex/tcolorbox/tcolorbox.doc.skins.tex#L2978

**Comment by T-F-S** [comment]
Well, this is not directly a typo, because I did it with intention as I have to admit. " ` " is so annoying to type with a german keyboard layout that I used " ' " everywhere...
Maybe, I should use `\enquote` oder `\enquote*` instead. I'm undecided yet.

By the way:
I would prefer not to have pull requests, because I use this repository as an export copy and I do not import from here.

**Comment by muzimuzhi** [comment]
fully understandable

**Comment by muzimuzhi** [comment]
3. sec 4.10 Height Control, `inherit height` option
![image](https://user-images.githubusercontent.com/6376638/90844209-f51a5f80-e395-11ea-8b5a-715169fa42e3.png)
It should be `/tcb/inherit height=<fraction>`, instead of just `=fraction`. Suggested change:

```diff
diff --git a/doc/latex/tcolorbox/tcolorbox.doc.coreoptions.tex b/doc/latex/tcolorbox/tcolorbox.doc.coreoptions.tex
index f053690..2965465 100644
--- a/doc/latex/tcolorbox/tcolorbox.doc.coreoptions.tex
+++ b/doc/latex/tcolorbox/tcolorbox.doc.coreoptions.tex
@@ -1809,7 +1809,7 @@ used for boxes inside a \refEnv{tcbraster}.


 \clearpage
-\begin{docTcbKey}[][doc new={2017-06-28}]{inherit height}{\colOpt{=fraction}}{default |1|, initially unset}
+\begin{docTcbKey}[][doc new={2017-06-28}]{inherit height}{\colOpt{=\meta{fraction}}}{default |1|, initially unset}
   If this option is used for a |tcolorbox| which is embedded inside
   another (outer) |tcolorbox| \emph{and} if this outer |tcolorbox| has
   a fixed height, then the given \meta{fraction} of the available text height
```

**Comment by muzimuzhi** [comment]
4. sec 4.14, option `force nobeforeafter` is documented as `forces nobeforeafter`, note the `s` in `forces`.

https://github.com/T-F-S/tcolorbox/blob/55d9deed444eef58c65a88b893500255f44ac7ba/doc/latex/tcolorbox/tcolorbox.doc.coreoptions.tex#L2869-L2874
https://github.com/T-F-S/tcolorbox/blob/55d9deed444eef58c65a88b893500255f44ac7ba/tex/latex/tcolorbox/tcolorbox.sty#L301

Related CHANGES lines
https://github.com/T-F-S/tcolorbox/blob/55d9deed444eef58c65a88b893500255f44ac7ba/doc/latex/tcolorbox/CHANGES#L737-L738

**Comment by T-F-S** [comment]
Thank you again - this is really a misleading documentation bug.

**Comment by T-F-S** [comment]
Resolved by version `4.32 (2020-09-17)`.
